clang/lib/CodeGen/CGLoopInfo.cpp

```diff
@@ -611,6 +611,14 @@ MDNode *LoopInfo::createMetadata(
                             llvm::Type::getInt32Ty(Ctx), VC.second))};
     LoopProperties.push_back(MDNode::get(Ctx, Vals));
   }
+
+  for (auto &FP : Attrs.SYCLIntelFPGANPipelines) {
```

Comment:
This may be a silly question: What does the 'N' after 'FPGA' stand for?

Reply:
Not sure why I added. Seems to me a copy-paste problem. Removed